db/dcrpg/internal/chartblockstmts.go

```diff
@@ -0,0 +1,25 @@
+package internal
+
+const (
+	// InsertChartBlock holds DML for Chart block data inserting.
+	InsertChartBlock = `INSERT INTO chartblocks VALUES ($1, $2, $3);`
```

Comment:
The work "chart" should not appear anywhere in this PR, except perhaps in a comment.  This PR needs to be serving a certain kind of data on one or more API endpoints, which may or may not be used for charting.